### DISCUSSION TITLE
Why are some apps in the hundreds of MB while other are only a few?

### DISCUSSION THREAD

**Discussion by project-guru** [dmarmor/epichrome#325] [General]
I have created 2 apps (both using brave) but one is about 483 MB while the other under 3 MB. See screenshot below.

I have a smallish SSD so I'm trying to use as little space as possible. 

Any idea how I can keep my apps small?

![2021-04-25_11-26-18](https://user-images.githubusercontent.com/18504294/116001303-6b243580-a5b9-11eb-874d-7f2b2ab76a8c.png)

**Comment by dmarmor** [comment]
So the first app is either running (most likely) or was left in an activated state after it quit (which can be resolved easily by running `Epichrome Scan.app`).

Please note that even though the app shows itself as being very large while running, it is _not_ using all that space. The entire framework folder, where most of the space is, is a hard link to the same folder in Epichrome itself, so even if you have a whole bunch of apps running, they're not using up any more space. You can confirm this by checking your SSD's capacity with no apps running, and then checking it again with several running. It should not have decreased much if at all.

**Reply by project-guru** [reply]
Oh yes! I quit the first app (seen on left side of the screenshot on my original post) and suddenly its size went down to 2.9 MB!  To clarify a couple of things however:

a) Are you saying that if for example, I was to run 10 or 20 apps they won't take up 10 or 20 times more hard drive space than if I ran just 1 app?

b) What exactly is the purpose of the Epichrome Scan.app? 

Thanks!

**Reply by dmarmor** [reply]
Thanks for the reply!

> a) Are you saying that if for example, I was to run 10 or 20 apps they won't take up 10 or 20 times more hard drive space than if I ran just 1 app?

Yes! That's exactly right. Each of those apps would appear to be very large as they're running, but you should not see your actual hard drive space decrease, because the engines are all just hard links to one engine framework.

> b) What exactly is the purpose of the Epichrome Scan.app?

`Epichrome Scan.app` exists to fix a problem that can occasionally occur where an app doesn't quit properly and is left in an activated state (in which case it would appear to be hundreds of MB). You can find out more about this and how to best deal with it in this [troubleshooting guide](https://github.com/dmarmor/epichrome/blob/master/TROUBLESHOOTING.md#opening-an-app-from-the-dock-opens-a-generic-browser).

**Reply by project-guru** [reply]
Great Thank you!
